src/pages/faq.tsx

```diff
@@ -18,50 +23,77 @@ interface faqElement {
   answer: string
 }
 
-export const getStaticProps = () => {
+interface Props {
+  faqJson: faqJson
+  serviceMessages: ServiceMessage[]
+}
+
+export const getServerSideProps = async () => {
+  const connection = getConnection()
+  const serviceMessagesResult = await getServiceMessages(connection, 0)
+  let serviceMessages: ServiceMessage[] = []
+  if (isError(serviceMessagesResult)) {
+    logger.error(serviceMessagesResult)
+  } else {
+    serviceMessages = JSON.parse(JSON.stringify(serviceMessagesResult.result))
+  }
+  console.log(serviceMessages)
```

Comment:
Do we need to be logging out to the console here?

Reply:
good spot, will remove